Xamarin/SSW.Consulting/SSW.Consulting.iOS/AppDelegate.cs

```diff
@@ -53,5 +54,14 @@ public override bool OpenUrl(UIApplication app, NSUrl url, NSDictionary options)
             AuthenticationContinuationHelper.SetAuthenticationContinuationEventArgs(url);
             return true;
         }*/
+        //public override void FailedToRegisterForRemoteNotifications(UIApplication application, NSError error)
```

Comment:
If this isn't being used then remove the comment. It'll be saved in the commit history